community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/birk/il/Instruction.scala

```diff
@@ -58,5 +64,7 @@ object Instruction {
     override def fields(): String = ""
 
     override def generateInit(): String = ""
+
+    override def exceptions: Set[ExceptionCodeGen] = Set.empty
```

Comment:
as above let's clean this up in a later PR, if we want to change this we should change all the methods in this trait